deploy/layered-sre-authorization/00-layered-sre-cluster-admins.Group.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+  name: layered-sre-cluster-admins
```

Comment:
Should we prefix this with osd, like our other groups?

Reply:
Went the layered-* route so we don't confuse with SRE P groups. layered-* will be added to the webhook validation.